mne/minimum_norm/resolution_matrix.py

```diff
@@ -0,0 +1,259 @@
+"""Coompute resolution matrix for linear estimators."""
+from copy import deepcopy
+
+import numpy as np
+
+from mne import pick_channels, EvokedArray
+from mne.utils import logger
+from mne.forward.forward import is_fixed_orient, convert_forward_solution
+
+from mne.minimum_norm import apply_inverse
+from mne.beamformer import make_lcmv
+
+
+def make_resolution_matrix(forward, inverse_operator, method, lambda2):
```

Comment:
use the same defaults as apply_inverse so you can just do

`make_resolution_matrix(forward, inverse_operator)`